Sources/swift-bundler/Bundler/ResourceBundler.swift

```diff
@@ -280,6 +284,10 @@ enum ResourceBundler {
         infoPlist =
           bundle
           .appendingPathComponent("Info.plist")
+      case .visionOS, .visionOSSimulator:
+        infoPlist =
+          bundle
+          .appendingPathComponent("Info.plist")
```

Comment:
Combine both of these cases into a single case (given that the logic is the same)